plugins/background_fetch/CMakeLists.txt

```diff
@@ -16,5 +16,5 @@
 #######################
 
 add_atsplugin(background_fetch background_fetch.cc configs.cc headers.cc rules.cc)
-target_link_libraries(background_fetch PRIVATE libswoc::libswoc ts::tsutil)
```

Comment:
Isn't the underlying issue that `background_fetch` depends on this? Maybe it could get all the needed utilities directly from libswoc?